ocaml/idl/datamodel_errors.ml

```diff
@@ -1968,6 +1968,9 @@ let _ =
   error Api_errors.host_evacuation_is_required ["host"]
     ~doc:"Host evacuation is required before applying updates." () ;
 
+  error Api_errors.vm_can_only_belong_to_one_group []
```

Comment:
How about `vm_can_only_belong_to_one_anti_affinity_group`?

Reply:
In the long run you could have affinity and anti-affinity groups and it could be legal for a VM to be a member of one each. Currently a VM can only be a member of either a normal or anti-affinity group. Does each VM has to be a member of exactly one group currently?

Reply:
> In the long run you could have affinity and anti-affinity groups and it could be legal for a VM to be a member of one each. Currently a VM can only be a member of either a normal or anti-affinity group. Does each VM has to be a member of exactly one group currently?

NO. A VM can belong to none of the groups.